parastell/magnet_coils.py

```diff
@@ -11,17 +13,46 @@
 export_allowed_kwargs = ["step_filename", "export_mesh", "mesh_filename"]
 
 
+def compute_tangent(prev_line, next_line):
+    """Computes tangent at "current" filament point using central difference
```

Comment:
```suggestion
    """Computes tangent at "current" filament point using forward difference
```

Reply:
I believe it is central difference since the approximation uses the next and previous points, rather than the forward difference which uses the next and current points.

Reply:
I see.... but this method actually just uses any two points, and it's your choice of which points to pass as arguments that makes a difference.  So technically, this method simply calculates a unit vector between two points.

Reply:
I suppose that's true. That said, the updated version I'm about to commit doesn't rely on this function anymore, so I've removed it from this script.